karma.conf.js

```diff
@@ -1,6 +1,85 @@
 const path = require('path');
 const webpackConfig = require('./webpack.config')[0];
 
+const USING_TRAVISCI = Boolean(process.env.TRAVIS);
+const USING_SL = Boolean(process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY);
+
+const SL_LAUNCHERS = {
+  'sl-chrome-stable': {
+    base: 'SauceLabs',
+    browserName: 'chrome',
+    version: '51'
```

Comment:
Ah! Yes!! That's exactly what I was looking for but I guess I was looking in the wrong place. That's so much better haha.